specs/engage-digital_openapi3.yaml

```diff
@@ -2081,6 +2081,22 @@ paths:
       summary: Updating an identity group
       tags:
         - Identity Groups
+  /interface_locales:
+    get:
+      description: This method renders all available locales for user interface.
+      operationId: getAllUiLocales
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                items:
+                  $ref: '#/components/schemas/UiLocale'
+                type: array
+          description: List of all locales

```

Comment:
No need to add "UI" in this description too? (and "content" in the other one)

Reply:
Running swaggman again doesn't update the postman?

Reply:
I added it

Reply:
ok, I guess it's a no for my second question :)